homeassistant/components/bang_olufsen/const.py

```diff
@@ -139,63 +139,63 @@ class WebsocketNotification(StrEnum):
     items=[
         Source(
             id="uriStreamer",
-            is_enabled=True,
+            is_enabled=True,  # type: ignore[call-arg]
             is_playable=True,
             name="Audio Streamer",
             type=SourceTypeEnum(value="uriStreamer"),
             is_seekable=False,
         ),
```

Comment:
The error is emitted for multiple attributes. I'd suggest to move it a bit higher, i.e.
```py
    Source(  # type: ignore[call-arg]
        id=...
```

Same for all other calls here.

Reply:
Adjusted in 2b47bdf206b